server/test/unit/server/webservices/views/test_permissions.py

```diff
@@ -4,7 +4,8 @@
 import mock
 
 from base import assert_auth_EXECUTE, assert_auth_READ
-from pulp.server.exceptions import InvalidValue
+from pulp.server.exceptions import MissingValue
+from pulp.server.webservices.views import permissions
```

Comment:
This import is somewhat redundant with the import on the next line. You could remove this line and import `_check_invalid_params` directly on line 9. I think that would be better.